Assets/Minigames/FrequencyMinigame/SineWaveController.cs

```diff
@@ -13,9 +13,11 @@ void Start()
     {
         frequencySlider.onValueChanged.AddListener((newFrequency) => {
             sineWave.frequency = newFrequency;
+            sineWave.DrawSineWave(amplitudeSlider.value, newFrequency);
```

Comment:
You have added the `amplitude` and `frequency` parameters to the `DrawSineWave` method, but you don't need to do that since you set the `sineWave`'s `frequency` member variable. I would therefore suggest removing the parameters from the method signature.